content/en/data_jobs/databricks.md

```diff
@@ -184,6 +184,16 @@ In Datadog, view the [Data Jobs Monitoring][6] page to see a list of all your Da
    }
    {{< /highlight >}}
 
+### Set up Data Jobs Monitoring with Databricks Networking Restrictions
+With the default [Databricks Networking Restrictions][12], Datadog may not have access to your Databricks APIs, which can prevent three key requirements for Data Jobs Monitoring from functioning properly:
+1. Ensure that the agent can be downloaded onto the Databricks cluster using the curl command in the init script.
```

Comment:
I think we don't need to list these (items 1-3), since my suggestion above gives an overview of the need for API connectivity.  The agent download and sending metrics/traces could technically be other firewall issues separate from Datadog's backend accessing customer Databricks APIs, and I think this should just be focused on the Databricks API access piece.

Reply:
Thanks, removed !